File: copy_img.py
import shutil
import os

src = "/Users/baljotchohan/BCA 2nd /BCA-2ND-SEM/Digital Empowerment .jpeg"
dst = "/Users/baljotchohan/BCA 2nd /BCA-2ND-SEM/study-dashboard/public/Digital_Empowerment.jpeg"

try:
    shutil.copy2(src, dst)
    print(f"Successfully copied {src} to {dst}")
except Exception as e:
    print(f"Error: {e}")

File: copy_img.js
const fs = require('fs');
const path = require('path');

const src = path.join(__dirname, 'MIS.jpeg');
const dst = path.join(__dirname, 'study-dashboard', 'public', 'MIS.jpeg');

try {
    fs.copyFileSync(src, dst);
    console.log(`Successfully copied ${src} to ${dst}`);
} catch (err) {
    console.error(`Error: ${err.message}`);
}
